app/client/ui/CustomWidgetGallery.ts

```diff
@@ -102,21 +102,10 @@ class CustomWidgetGallery extends Disposable {
       }
     });
 
-    this._saveDisabled = Computed.create(this, use => {
-      const selectedWidgetId = use(this._selectedWidgetId);
-      if (!selectedWidgetId) { return true; }
-      if (!this._section) { return false; }
-
-      const savedWidgetId = use(this._savedWidgetId);
-      if (selectedWidgetId === CUSTOM_URL_WIDGET_ID) {
-        return (
-          use(this._savedWidgetId) === CUSTOM_URL_WIDGET_ID &&
-          use(this._customUrl) === use(this._section.customDef.url)
-        );
-      } else {
-        return selectedWidgetId === savedWidgetId;
-      }
-    });
+    this._saveDisabled = Computed.create(
```

Comment:
Can you expand it? I think it is best to do explicit ifs and returns of true/false in such contexts as here. Otherwise it is impossible to read; I'm too old to spend my life figuring out what this means:
the save is disabled when negation of truthiness of selectedWidget is true? 🤯

Also the comment from PR description can be put here.